tools/mksurfdata_esmf/README

```diff
@@ -53,9 +53,31 @@ and able to be referenced with the env variable PIO which will need to be in
 the porting instructions for the machine. Currently an independent PIO library
 is not available on cime ported machines.
 
-=======================
+=================================================
+IMPORTANT NOTE: ONLY WORKING ON DERECHO CURRENTLY
+=================================================
+
+Currently we have only tested and ran on Derecho. In the longer
+term we will make sure the process works for other CESM machines.
+
+Here's one issue tracking progress for this.
```

Comment:
It feels safer not to promise that we will make sure it works for other machines. Here is my suggestion:

```suggestion
Currently we have run and tested mksurfdata_esmf on Derecho. Please see this github issue about mksurfdata_esmf on other CESM machines:
```